packages/react-instantsearch/src/core/indexUtils.js

```diff
@@ -1,4 +1,4 @@
-import { has, omit, get } from 'lodash';
+import { has, omit, get, isObject } from 'lodash';
```

Comment:
I believe that because of prettier we don't have this eslint rules anymore? That's annoying, do you think you can have a look about it? I will fix this.